Gems/Atom/Tools/MaterialEditor/Code/Source/MaterialEditorApplication.cpp

```diff
@@ -71,8 +71,8 @@ namespace MaterialEditor
     }
 
     MaterialEditorApplication::MaterialEditorApplication(int* argc, char*** argv)
```

Comment:
The timer setup code can move to the base class 

connect(&m_timer, &QTimer::timeout, this, [&]()
        {
            this->PumpSystemEventLoopUntilEmpty();
            this->Tick();
        });